accesscontroltool-bundle/src/main/java/biz/netcentric/cq/tools/actool/authorizableinstaller/impl/AuthorizableInstallerServiceImpl.java

```diff
@@ -65,6 +66,7 @@
 import biz.netcentric.cq.tools.actool.configmodel.pkcs.Key;
 import biz.netcentric.cq.tools.actool.configmodel.pkcs.RandomPassword;
 import biz.netcentric.cq.tools.actool.crypto.DecryptionService;
+import biz.netcentric.cq.tools.actool.externalusermanagement.ExternalGroupManagement;
```

Comment:
the code in general is a bit inconsistent regarding the naming user vs group management. 

To the AC tool is 99% used in AEM (while pure sling is also supported) - we could call it just ImsGroupManagement for the 99% or we stay generic for the 1%. But I'm fine to stay generic for the 1%, but `user` is confusing because it is really only touching groups in IMS. So I would go for `ExternalGroupManagement` consistently.

Reply:
thinking a bit more, maybe the best package name is `idpintegration`

Reply:
user management for me comprises both user and group management (also the API from Adobe is called User Management API, UMAPI). IDP for me is only the authentication part but not the management part (compare with https://www.adobe.com/content/dam/cc/en/trust-center/ungated/whitepapers/corporate/adobe-identity-management-services-security-overview.pdf)

Reply:
The idea was to add user management class in the same package (in order to export users if there is the need in the future).